tools/unencrypted-keys-gen/main.go

```diff
@@ -18,11 +18,13 @@ var (
 	overwrite  = flag.Bool("overwrite", false, "If the key file exists, it will be overwritten")
```

Comment:
Should this be file be using the deterministic keys gen?

Reply:
Yes, but that's unrelated to this `start.sh` support script in this PR. This file was only edited to make the structs public for translating the upstream yaml to json that we can consume.